### PR TITLE
Add Raisin

### PR DESCRIPTION
**Pull request by contextnerror**
Screen Shot 2023-08-01 at 8 31 50 AM

### CODE REVIEW

File: entries/r/raisin.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Raisin": {
+    "domain": "raisin.com",
```
Comment (hkamran80) [inline entries/r/raisin.com.json:3]: Do you know if Raisin's regional sites share a common authentication backend?

File: img/r/raisin.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 284.33 284.33"><defs><linearGradient id="a" x1="43.45" y1="240.88" x2="242.69" y2="41.65" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#156cc4"/><stop offset=".16" stop-color="#1367bb"/><stop offset=".41" stop-color="#0e5ba2"/><stop offset=".73" stop-color="#07487b"/><stop offset="1" stop-color="#003554"/></linearGradient></defs><path d="M142.16.01C63.65.01 0 63.66 0 142.18c0 29.51 8.99 56.92 24.39 79.64l79.65-79.65h-66.2V88.24H196.1V246.5h-53.93v-66.2L109 213.47l-46.47 46.47c22.72 15.39 50.13 24.39 79.64 24.39 78.51 0 142.16-63.65 142.16-142.16S220.68.01 142.16.01Z" fill="url(#a)"/></svg>
```
Comment (hkamran80) [inline img/r/raisin.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 284.33 284.33"><defs><linearGradient id="a" x1="43.45" x2="242.69" y1="240.88" y2="41.65" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#156cc4"/><stop offset=".16" stop-color="#1367bb"/><stop offset=".41" stop-color="#0e5ba2"/><stop offset=".73" stop-color="#07487b"/><stop offset="1" stop-color="#003554"/></linearGradient></defs><path fill="url(#a)" d="M142.16.01C63.65.01 0 63.66 0 142.18c0 29.51 8.99 56.92 24.39 79.64l79.65-79.65h-66.2V88.24H196.1V246.5h-53.93v-66.2L109 213.47l-46.47 46.47a141.53 141.53 0 0 0 79.64 24.39c78.51 0 142.16-63.65 142.16-142.16S220.68.01 142.16.01Z"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/r/raisin.com.json (hunk 1 above)
Comment (contextnerror) [reply entries/r/raisin.com.json:3]: No, I don't know.

Comment (hkamran80) [inline entries/r/raisin.com.json:3]: Let's work under the assumption that they do share an authentication backend.

```suggestion
    "domain": "raisin.com",
    "additional-domains": [
      "weltsparen.at",
      "weltsparen.de",
      "raisin.nl",
      "raisin.es",
      "raisin.fr",
      "raisin.ie",
      "raisin.co.uk"
    ],
```

File: entries/r/raisin.com.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "tfa": [
+      "sms",
+      "call",
+      "totp"
+    ],
+    "categories": [
+      "banking"
+    ]
```
Comment (hkamran80) [inline entries/r/raisin.com.json:51]:
```suggestion
    ],
    "regions": [
      "at",
      "be",
      "bg",
      "cy",
      "cz",
      "dk",
      "ee",
      "fi",
      "fr",
      "de",
      "gr",
      "hr",
      "hu",
      "ie",
      "it",
      "lv",
      "lt",
      "lu",
      "mt",
      "nl",
      "pl",
      "pt",
      "ro",
      "sk",
      "si",
      "es",
      "se",
      "us",
      "gb"
    ]
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**